llvm/lib/LTO/LTOBackend.cpp

```diff
@@ -201,7 +201,12 @@ static std::unique_ptr<TargetMachine>
 createTargetMachine(const Config &Conf, const Target *TheTarget, Module &M) {
   StringRef TheTriple = M.getTargetTriple();
   SubtargetFeatures Features;
-  Features.getDefaultSubtargetFeatures(Triple(TheTriple));
+  StringRef TargetABI = "";
+  if (auto TargetABIMD =
```

Comment:
I think this should be `auto *TargetABIMD`. Coding standards say to still make it obvious when something is a pointer.